apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -124,26 +121,23 @@ const styles = {
     minHeight: 200,
   },
   instructions: {
-    padding: '5px 0'
-  },
-  instructionsChatBubble: {
-    backgroundColor: 'white',
-    borderRadius: 10,
-    marginBottom: 10,
-    padding: 10,
-  },
-  instructionsChatText: {
-    // note that we actually do need to specify lineHeight
-    // in pixels; it defaults to a multiple of fontSize
-    lineHeight: '25px',
-    fontSize: 16,
-    color: 'black',
-    margin: 0
+    padding: '5px 0',
+  },
+  instructionsWithTips: {
+    width: 'calc(100% - 20px)',
+    float: 'right'
   },
 };
 
 var TopInstructions = React.createClass({
```

Comment:
Is there any code in this JSX file that can be shared with https://github.com/code-dot-org/code-dot-org/commits/staging/apps/src/templates/instructions/TopInstructionsCSP.jsx?